### PR TITLE
kamel 2.2.0

### PR DESCRIPTION
**Pull request by gansheer**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by gansheer** [comment]
Should replace invalid PR #159288

If another procedure should be applied in bot test failure case don't hesitate to tell me @chenrui333 :pray: .

**Comment by SMillerDev** [comment]
> Should replace invalid PR https://github.com/Homebrew/homebrew-core/pull/159288

Why is that one invalid?

**Comment by p-linnane** [comment]
Please squash your commits into one with the message `kamel 2.2.0`.

**Comment by gansheer** [comment]
> > Should replace invalid PR #159288
> 
> Why is that one invalid?

The behavior of the command changed, therefore the test should as well.

File: Formula/k/kamel.rb [hunk 1]
```diff
@@ -44,7 +44,9 @@ def install
     assert_match version.to_s, version_output
 
     run_output = shell_output("echo $(#{bin}/kamel run 2>&1)")
-    assert_match "Error: run expects at least 1 argument, received 0", run_output
+    assert_match "Error: " \
+                 "invalid configuration: " \
+                 "no configuration has been provided, try setting KUBERNETES_MASTER environment variable", run_output
```
Comment (chenrui333) [inline Formula/k/kamel.rb:49]: I just removed this test in https://github.com/Homebrew/homebrew-core/pull/159288

Comment (gansheer) [reply Formula/k/kamel.rb:49]: Should the test just stay removed or should i keep the current PR opened to follow up the #159288 after it is merged ?

Comment (chenrui333) [reply Formula/k/kamel.rb:49]: this pr would be closed by https://github.com/Homebrew/homebrew-core/pull/159288 per merge

**Pull request closed without merging**